exercises/ledger/Example.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+
+public class LedgerEntry
+{
+    public LedgerEntry(DateTime date, string description, float change)
+    {
+        Date = date;
+        Description = description;
+        Change = change;
+    }
+
+    public DateTime Date { get; }
+    public string Description { get; }
+    public float Change { get; }
+}
+
+public static class Ledger
+{
+    private const int TruncateLength = 25;
+    private const string TruncateSuffix = "...";
+
+    public static LedgerEntry CreateEntry(string date, string description, int change) =>
+        new LedgerEntry(ParseDate(date), description, ParseChange(change));
+
+    private static DateTime ParseDate(string date) => DateTime.Parse(date, System.Globalization.CultureInfo.InvariantCulture);
+
+    private static float ParseChange(int change) => change / 100.0f;
+
+    private static string CurrencySymbol(string currency)
+    {
+        switch (currency)
+        {
+            case "USD": return "$";
+            case "EUR": return "€";
+            default: throw new ArgumentException("Invalid currency");
+        }
+    }
+
+    private static CultureInfo CultureInfo(string locale)
+    {
+        switch (locale)
+        {
+            case "en-US": return new CultureInfo(locale);
+            case "nl-NL": return new CultureInfo(locale);
+            default: throw new ArgumentException("Invalid locale");
+        }
+    }
+
+    private static string ShortDateFormat(string locale)
```

Comment:
I knew the short date format that you provided for `en-US` was the default but wasn't sure for `nl-NL` which I tested and learned is actually `16-1-2017` for today's date. You learn something new everyday :)

https://dotnetfiddle.net/flhHLk